gnd/src/main/java/com/google/android/gnd/persistence/remote/firestore/schema/FeatureDocument.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.gnd.persistence.remote.firestore.schema;
+
+import androidx.annotation.Nullable;
+import com.google.android.gnd.persistence.remote.firestore.AuditInfoDoc;
+import com.google.firebase.firestore.GeoPoint;
+import com.google.firebase.firestore.IgnoreExtraProperties;
+
+/** Feature entity stored in Firestore. */
+@IgnoreExtraProperties
+class FeatureDocument {
+  @Nullable private String featureTypeId;
```

Comment:
Is it possible for us to rename this `layerId` at this point--or are we still blocked on that for compatibility reasons?

Reply:
It's here for compatibility with the current web console app. Once the Angular migration is done we can switch the Android client to the new schema. In theory we could do it now, but it would make it harder to set up and edit projects.